Rewind param: stop making the non-ably-sdk fallback method the first thing we tell people about (#3334)
ably/docs: src/pages/docs/channels/options/rewind.mdx

```
</Code>

<Aside data-type="note">

> If using [MQTT](/docs/protocols/mqtt) or [SSE](/docs/protocols/sse), see [Channel options without Ably SDK support](/docs/channels/options#other-sdks) for how to specify a rewind param.

Reviewer: ```suggestion
If using [MQTT](/docs/protocols/mqtt) or [SSE](/docs/protocols/sse), you can [specify channel params](/docs/channels/options#other-sdks) as a query string.
```
